src/components/inputs/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import styles from './Checkbox.module.scss';
+import {
+  Checkbox as CheckboxAria,
+  type CheckboxProps as CheckboxPropsAria
+} from 'react-aria-components';
+
+type CheckBoxProps = Omit<CheckboxPropsAria, 'children'> & {
+  children?: React.ReactNode;
+  variant?: 'default' | 'secondary';
+};
+
+export const Checkbox: React.FC<CheckBoxProps> = ({ children, variant = 'default', ...props }) => {
+  const rootClassname = classNames(styles['react-aria-Checkbox'], styles[variant]);
+  return (
+    <CheckboxAria {...props} className={rootClassname}>
+      {({ isIndeterminate }) => (
+        <>
+          <div className={`${styles['checkbox']} ${styles[variant]}`}>
```

Comment:
variant has been moved up :)
```suggestion
          <div className={`${styles['checkbox']}`}>
```